webapp/src/styles.scss

```diff
@@ -61,6 +61,26 @@ input[type=text].default-input, input:not([type]).default-input {
 	outline: solid 1px mat-color($mat-blue, 500);
 }
 
+
+.default-input.mat-select {
```

Comment:
Where are these styles used?

Reply:
They are used for selects inside widgets. Without these there are no borders and the size is slightly messed up. See https://github.com/CCDirectLink/crosscode-map-editor/blob/enemy/webapp/src/app/shared/widgets/string-widget/string-widget.component.html#L4-L10

Reply:
alright